test/scripts/jenkins_build_kibana.sh

```diff
@@ -19,5 +19,5 @@ yarn run grunt functionalTests:ensureAllTestsInCiGroup;
 # Do not build kibana for code coverage run
 if [[ -z "$CODE_COVERAGE" ]] ; then
   echo " -> building and extracting OSS Kibana distributable for use in functional tests"
-  node scripts/build --debug --oss
+  node scripts/build --debug --oss --all-platforms
```

Comment:
Why build for all platforms?

Reply:
I'll revert before merging, hack to grab deb/rpm uploads

Reply:
Cool, draft mode would be preferred